src/excmds.ts

```diff
@@ -311,6 +311,20 @@ export function zoom(level=0){
     browser.tabs.setZoom(level)
 }
 
+//#background
+export async function reader() {
+    let aTab = await activeTab()
+    if (aTab.isArticle) {
+	// need tabs as any, since typescript believes that:
+	// Property 'toggleReaderMode' does not exist on type 'typeof tabs'.
+	// (Though actually, the code transpiles correctly...)
+	(browser.tabs as any).toggleReaderMode()
```

Comment:
The preferred way of doing this is to add some lines to tridactyl.d.ts and add the function there.

Every now and then we'll merge it upstream into web-ext-types.